src/sentry/utils/performance_issues/detectors/consecutive_db_detector.py

```diff
@@ -216,7 +216,7 @@ def is_event_eligible(cls, event, project: Project = None) -> bool:
 
         if request:
             url = request.get("url", "") or ""
-            method = request.get("method", "") or ""
+            method = request.get("http.method", "") or request.get("method", "") or ""
```

Comment:
I'd leave a TODO w/ a cleanup note here that `method` can be removed likely in 6 month+ when the sdk has been more adopted.